src/main/java/com/eternalcode/discordapp/leveling/leaderboard/LeaderboardService.java

```diff
@@ -0,0 +1,42 @@
+package com.eternalcode.discordapp.leveling.leaderboard;
+
+import com.eternalcode.discordapp.leveling.Level;
+import com.eternalcode.discordapp.leveling.LevelService;
+import net.dv8tion.jda.api.EmbedBuilder;
+
+import java.awt.Color;
+import java.util.List;
+
+public class LeaderboardService {
+
+    private static final int PAGE_SIZE = 10;
+
+    private final LeaderboardConfiguration leaderboardConfiguration;
+    private final LevelService levelService;
+
+    public LeaderboardService(LeaderboardConfiguration leaderboardConfiguration, LevelService levelService) {
+        this.leaderboardConfiguration = leaderboardConfiguration;
+        this.levelService = levelService;
+    }
+
+    public List<Level> getLeaderboard(int start, int end) {
+        int pageSize = end - start;
+
+        return this.levelService.getTop(pageSize, start).join();
+    }
+
+    public EmbedBuilder createEmbedBuilder(int currentPage, int totalPages) {
+        return new net.dv8tion.jda.api.EmbedBuilder()
```

Comment:
Tutaj można użyć importu.
```suggestion
        return new EmbedBuilder()
```

Reply:
Fakt, coś mi ta intelka namieszała